scripts/process_jartic_parallel.py

```
@@ -0,0 +1,320 @@
+                                    import os
+                                    prefecture, temp_path, record_count = future.result(timeout=300)
+                                    
+                                    if temp_path and record_count > 0:
+                                        if not header_written:
+                                            writer = csv.writer(outfile)
+                                            writer.writerow(['timestamp', 'source_code', 'point_number', 'point_name', 
+                                                           'mesh_code', 'link_type', 'link_number', 'traffic_volume', 
+                                                           'distance', 'version', 'prefecture'])
+                                            header_written = True
+                                        
+                                        with open(temp_path, 'r', encoding='utf-8') as temp_file:
```

Ruff (0.12.2)

Unnecessary mode argument

Remove mode argument

(UP015)